index.bs

```diff
@@ -3114,11 +3119,22 @@ A <dfn>viewport configuration</dfn> is a [=struct=] with an [=struct/item=] name
 <dfn attribute for="viewport-configuration">viewport</dfn> which is a [=viewport-configuration/viewport dimensions=]
 or null and an [=struct/item=] named <dfn attribute for="viewport-configuration">devicePixelRatio</dfn> which is a float or null.
 
+An <dfn>unhandled prompt behavior</dfn> is a [=struct=] with:
+* [=struct/Item=] named <dfn attribute for="unhandled-prompt-behavior-alert">alert</dfn> which is a string or null (default: null);
```

Comment:
Doesn't `or null` already indicate that it is the default. At least that is what we are using for the other override maps.

Also I wonder if it should actually have values of kind struct with `handler` and `notify`. I assume we do not want to have to deserialize the behavior each time when we access this map.

Reply:
This struct is used to store the overrides and does not allow for classics string values like  "dismiss and notify".